manageiq-operator/pkg/helpers/miq-components/cr.go

```diff
@@ -97,6 +97,22 @@ func httpdImageTag(cr *miqv1alpha1.ManageIQ) string {
 	}
 }
 
+func httpdImage(cr *miqv1alpha1.ManageIQ, privileged bool) string {
+	if cr.Spec.HttpdImage == "" {
+		var image string
+
+		if privileged {
+			image = "httpd-init"
+		} else {
+			image = "httpd"
+		}
+
+		return cr.Spec.HttpdImageNamespace + "/" + image + ":" + cr.Spec.HttpdImageTag
+	} else {
+		return cr.Spec.HttpdImage
+	}
+}
```

Comment:
I feel like this would be cleaner if written in guard clause style:

```suggestion
	if cr.Spec.HttpdImage != "" {
		return cr.Spec.HttpdImage
	}
		
	var image string

	if privileged {
		image = "httpd-init"
	} else {
		image = "httpd"
	}

	return cr.Spec.HttpdImageNamespace + "/" + image + ":" + cr.Spec.HttpdImageTag
}
```

Reply:
Same comment for all of the other places as well.

Reply:
👍  will update for httpd because of added logic, the others follow the pattern of all others in that file, so I prefer to keep them the same for consistency.